javascript/src/main/services/scheduleAPI.js

```diff
@@ -0,0 +1,81 @@
+import { fetchWithToken } from "main/utils/fetch"
+
+
+
+//schedule
+const fetchcreateScheduleJSON = async (schedule, getToken, onSuccess, onError) => {
+    const token = await getToken();
+    const url = "/api/public/createSchedule?" + new URLSearchParams(schedule);
+    try {
+        const json = await fetchWithToken(url, getToken, {
+            method: "POST",
+            headers: {
+                Authorization: `Bearer ${token}`,
```

Comment:
Is this necessary (in this function and in the other functions in this file)? Doesn't `fetchWithToken` handle this for us?

Reply:
I believe you are correct, I removed the lines and retested everything and it looks like it should all be in order now. Thanks for the input!

Reply:
I agree with @btk5h ... take a look at the implementation of `fetchWithToken` and make sure you are not reinventing the wheel.